src/main/java/com/school/project/controller/UserController.java

```diff
@@ -21,9 +23,30 @@
     @Autowired
     UserService userService;
 
+    @Autowired
+    UserAccountService userAccountService;
+
     @PostMapping("/users")
-    public void createUser(@RequestBody UserDto userDto) {
-        userService.create(convertUserDtoToUser(userDto));
+    public ResponseEntity<UserDto> createUser(@RequestBody UserDto userDto) {
+        return ResponseEntity.ok()
+                .body(convertUserToUserDto(userService
+                        .create(convertUserDtoToUser(userDto))));
+    }
+
+    @PutMapping("/users/userAccounts/{id}")
+    public ResponseEntity updateUserAccountType(@PathVariable long id, @RequestBody int accountType){
+        userAccountService.getUserAccountByUserId(id).setAccountRole(chooseType(accountType));
```

Comment:
writing here to the second time: you are not supposed to call userAccountService in UserController. Please read all my comment more carefully.